Dockerfilebck

```diff
@@ -0,0 +1,38 @@
+ARG RUBY_IMAGE="ruby:2.7.3-alpine"
```

Comment:
is this file added intentionally?

Reply:
Oh dear! Good spot! I somehow used `git add .` in a uncleaned workspace.